phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -718,9 +731,9 @@ private PTable addDerivedColumnsFromAncestors(PTable table, long timestamp,
         List<PColumn> excludedColumns = Lists.newArrayList();
```

Comment:
These are for primary key columns present in the base table, but dropped in the view, right? Let's rename this to `excludedPkColumns`